features/facebookImport/src/components/buttons/routeButton.jsx

```diff
@@ -9,21 +9,16 @@ const RouteButton = ({
     children,
     onClick = () => {},
 }) => {
-    const { handleBack, changeNavigationState, navigationState } =
-        useContext(ImporterContext);
     const history = useHistory();
 
-    let changedNavigationState = navigationState;
+    const newHistoryState = { ...INITIAL_HISTORY_STATE, ...stateChange };
```

Comment:
Wouldn't it be convenient to define a ScrollingState class, so that the attribute scrollingProgress is clear for everyone? Because essentially what you're doing here with the spread operator is to substitute the value of scrollintProgress in INITIAL_HISTORY_STATE with whatever value is in stateChange, if any, is that correct?

Reply:
Gives the button the power to "change screen and start at scrolling 200" fro example. So it's always able to overwrite the default

Reply:
But that's equivalent to declaring a class with a default value for an attribute, right?

Reply:
Totally, except you have a class :D